Tools/WinMLRunner/src/Run.cpp

```diff
@@ -290,19 +290,15 @@ HRESULT CreateSession(LearningModelSession& session, IDirect3DDevice& winrtDevic
                      // If DXGI factory creation was successful then get the IDXGIAdapter from the LUID acquired from
                      // the selectedAdapter
                      std::cout << "Using DXGI for adapter creation.." << std::endl;
+                     size_t luidSize;
```

Comment:
initialize luidSize to 0?

Reply:
You know, looking at this I'm realizing I should really be using the templated version of GetProperty that doesn't accept a size parameter for properties that aren't variable length.